test/test_cuda.py

```diff
@@ -292,6 +292,24 @@ def test_memory_allocation(self):
                 torch.cuda.caching_allocator_delete(mem)
                 self.assertEqual(torch.cuda.memory_allocated(), prev)
 
+    def test_get_alloc_free_events(self):
+        gc.collect()
+        size = 1
+        ptr = 0
+        mem = torch.cuda.caching_allocator_alloc(size)
+        current_device = torch.cuda.current_device()
+        events = torch.cuda.get_alloc_free_events()[current_device]
+        num_events = len(events)
+        ptr = events[-1].ptr
+        self.assertTrue(events[-1].size > 0)
```

Comment:
any reason for testing >0 instead of ==1? When will we get a different size?

Reply:
The returned size is actually rounded up to 512. I can test for this.

Reply:
Done. Thank you. It was because of previous implementation returned the actual size of allocated block which can be different.